source/deployment/advanced-permissions.rst

```diff
@@ -56,16 +56,27 @@ The interface for naming, assigning teams, and editing permissions in a Team Ove
 
 .. image:: ../images/team-scheme.png
 
-Channel Override Permissions (E20)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Channel Permissions 
+---------------------------
 
-*Available in a future release of Enterprise Edition E20*
+Channel Moderation (Beta) (E20)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*Available in Enterprise Edition E20*
+
+From v5.22, the channel moderation settings allow Admins to restrict actions within specific channels. These actions include: 
+
+- **Make channel read-only:** The ability for Admins to turn off posting in specified channels.
+- **Restrict reactions:** Turn off the ability for members and guests to post reactions.
+- **Restrict channel mentions:** Turn off the ability for users to post channel wide mentions (@all/channel/here) in specified channels.
+- **Channel member management:** Only Admins have the ability to add and remove channel members in the specified channels.
+
+These settings are modified in **System Console > User Management > Channels**. 
 
-Allows Admins to restrict permissions within specific channels. Permissions under consideration for this phase include:
+**Note:**
+These settings are applicable only to Guests and Members. System, Team, and Channel Admins are not affected. 
```

Comment:
I would prefer this have some guidance to better communicate that designating "announcer" or similar role in a read only channel is done by promoting the user to channel admin. I am not sure that is clear. 

suggested: If you wish to grant posting ability to a single member, you must promote that member to channel admin.